NTupleAnalyzer_nanoAOD.py

```diff
@@ -1324,7 +1388,7 @@ def TightHighPtIDMuons(T,_met,variation,isdata):
 			NewMu = TLorentzVector()
 			OldMu = TLorentzVector()
 			NewMu.SetPtEtaPhiM(_MuonCocktailPt[n],_MuonCocktailEta[n],_MuonCocktailPhi[n],0)
-			OldMu.SetPtEtaPhiM(T.Muon_tunepRelPt[n]*T.Muon_pt[n],T.Muon_eta[n],T.Muon_phi[n],0)#fixme cocktail pt?
+			OldMu.SetPtEtaPhiM(_preSmearCocktailPt[n], _preSmearCocktailEta[n], _preSmearCocktailPhi[n],0)#fixme cocktail pt?
```

Comment:
Here there might be an issue. You set preSmear quantities before the TuneP pt reordering. Should the preSmear quantities be set after the TuneP reordering, but before the smearing? Otherwise you could be comparing different muons.